src/prettytable/__init__.py

```diff
@@ -27,18 +30,6 @@
 )
 
 __all__ = [
-    "ALL",
-    "DEFAULT",
-    "DOUBLE_BORDER",
-    "SINGLE_BORDER",
-    "FRAME",
-    "HEADER",
-    "MARKDOWN",
-    "MSWORD_FRIENDLY",
-    "NONE",
-    "ORGMODE",
-    "PLAIN_COLUMNS",
-    "RANDOM",
```

Comment:
I removed the constants from `__all__` so type checks would emit warnings when there are used. The import is still possible though as to not break existing code.